docs/selection-methods.md

```diff
@@ -7,40 +7,104 @@ HyPhy provides a suite of diverse phylogenetic methodologies for testing specifi
+ 
+
+**If you use MEME in your analysis, please cite the following:** [`Murrell, B et al. "Detecting individual sites subject to episodic diversifying selection." PLoS Genetics 8, e1002764 (2012).`](http://dx.doi.org/10.1371/journal.pgen.1002764)
 
 <!--------------------------------------------------------------------------------------->
 ## aBSREL
 
+aBSREL ("adaptive Branch-Site Random Effects Likelihood) is an "adaptive" version of the commonly-used "branch-site" models, which are used to test if positive selection at specific sites has occurred on a proportion of branches. As such, aBSREL models both site-level and branch-level ω heterogeneity.
+
+aBSREL differs from other branch-site implementations by inferring the optimal number of ω categories for each branch. For example, the earlier HyPhy branch-site approach (BS-REL) assumed three ω rate categories for each branch and assigned each site, with some probability, to one of these categories. aBSREL, by contrast, acknowledges that different branches may feature more or less complex evolutionary patterns and hence may be better modeled by more or fewer ω categories. aBSREL specifically uses AIC<sub>c</sub> (small sample AIC) to infer the optimal number of rate categories for each branch. 
+
+After aBSREL fits the full adaptive model, hypothesis testing is performed for each branch by comparing, using the Likelihood Ratio Test, the full model to a null model where branches are not allowed to have categories of ω>1. 
+
+Most importantly, aBSREL can be run in two modes:
+
+* Select a set of "foreground" branches to explicitly test for positive selection. 
+* Perform an exploratory analysis where all branches are tested for positive selection. In this scenario, p-values at each branch must be corrected for multiple testing (using the Holh-Bonferroni correction). Due to the increase in multiple testing, this approach *has much lower power* compared to the former approach. 
+
+
+**If you use aBSREL in your analysis, please cite the following:** [`Smith, MD et al. "Less is more: an adaptive branch-site random effects model for efficient detection of episodic diversifying selection." Mol. Biol. Evol. 32, 1342–1353 (2015).`](https://doi.org/:10.1093/molbev/msv022)
+
 
 <!--------------------------------------------------------------------------------------->
 ## BUSTED
 
+Fits two models (con, uncon). Models can be fit either to the entire data set, or to specific sites. Hypothesis testing performed across whole data, whereas evidence ratios (which give **Descriptive, ad hoc** evidence of positive selection at individual sites, not a p-value) can be inferred on a whole-gene or per-site basis. On per-site, the third omega category is 
```

Comment:
Stubbing out for constrained/unconstrained. This is a placeholder, stay tuned for actual description..